src/api/app/models/bs_request.rb

```diff
@@ -107,6 +108,20 @@ def check_creator
     errors.add(:creator, "Login #{user.login} is not an active user")
   end
 
+  def double_actions
```

Comment:
Well you compare two things if they are the same. Which is the definition of [duplicate](https://www.merriam-webster.com/dictionary/duplicate).